lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextManager.cs

```diff
@@ -33,18 +34,24 @@ namespace Org.Apache.REEF.Common.Runtime.Evaluator.Context
 {
     public sealed class ContextManager : IDisposable
     {
-        private static readonly Logger LOGGER = Logger.GetLogger(typeof(ContextManager));
+        private static readonly Logger Logger = Logger.GetLogger(typeof(ContextManager));
```

Comment:
This change seems unrelated to the goal at hand. Is this part of new style guides?